src/common/options/global.yaml.in

```diff
@@ -3873,7 +3873,7 @@ options:
 - name: memstore_device_bytes
   type: size
   level: advanced
-  default: 1_G
+  default: 20_G
```

Comment:
We probably can't change this default.

Reply:
that is just for my test, not from this PR, this PR only included the last 3 commits.  I will rebase it after yingxin's PR get merged.